docs/examples/how_to_auxiliary.rst

```diff
@@ -90,7 +90,7 @@ to be implemented:
 
 - :py:meth:`~pykiso.interfaces.thread_auxiliary.AuxiliaryInterface._create_auxiliary_instance`:
   handle the auxiliary creation. Minimal actions to perform are
-  opening the attached :py:class:`~pykiso.connector.CChannel`, to which can be added actions such as flashing the device under test,
+  opening the attached :py:class:`~pykiso.connector.CChannel` with `self.my_callback`, to which can be added actions such as flashing the device under test,
```

Comment:
Currently the callback doesn't know in which context its called for example  open, close or receive. this could be an argument.

Reply:
I do not understand what you want to say :-/ The callback will always be called in the context of the person calling it. In our case:
* calling auxiliary if sync approach (all what I have right now)
* in an "Interruption" or "signal" if the connector forward to the driver it initialises a callback.

We could change the name to something like "my_notifyer" to be more precise. In reality, it does not matter in which context it is called, it matters more that it is called when something is happening which is relevant for the auxiliary.